main.go

```diff
@@ -105,6 +107,21 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "LVMCluster")
 		os.Exit(1)
 	}
+	// TODO only run when not in SNO
```

Comment:
Lets not leave any todos in the code

Reply:
This is depending on https://github.com/openshift/lvm-operator/pull/370 because we need the SNO check here to start / not start the controller (if we are in SNO the controller is not needed because the only master will never be removed without the rest of the cluster).

/hold because of this